llvm/lib/MCCAS/MCCASObjectV1.cpp

```diff
@@ -898,11 +898,20 @@ materializeDebugLineSection(MCCASReader &Reader,
   uint8_t OpcodeBase = 0;
   SmallVector<char, 0> DebugLineSection;
   bool DistinctDebugLineRefSeen = false;
+  bool DebugLineUnoptRefSeen = false;
   for (auto Ref : Refs) {
     auto Node = Reader.getObjectProxy(Ref);
     if (!Node)
       return Node.takeError();
```

Comment:
Why can't the other blocks be

```
if () {
// block specific code
} else if {
// block specific code
}
```

Rather than

```
if () {
// block specific code
  continue;
  }
  if () {
// block specific code
  continue;
  }
  ```
  
  Either way is fine by me but I am curious if llvm prefers one or the other?

Reply:
> llvm prefers one or the other?

This particular situation is not covered in the programming guidelines. While the `PaddingRef` one has more objective reasons to use a continue, I suggested doing the same for the other cases purely from personal preference. If we `continue`, we signal to the readers that we're done handling that case, so the readers don't need to worry whether there will be more common code at the end of the if/else chain that impacts that block of code.

Reply:
That is fine with me, I think consistency is a good thing